### PR TITLE
Remove obsolete e-smith fail2ban database

### PR DESCRIPTION
**Pull request by stephdl**
The fail2ban esmith db is no longer used.
But it may still be found in systems and can be quite large.
nethserver-inventory sends all esmith dbs.
If the unused fail2ban db is big the inventory is not stored in my.nethesis.it.

https://github.com/NethServer/dev/issues/6370

### CODE REVIEW

File: createlinks [hunk 1]
```diff
@@ -7,7 +7,8 @@ my $event = 'nethserver-fail2ban-update';
 event_actions($event,
              'initialize-default-databases' => '00',
              'nethserver-fail2ban-clean-sqlite-backup' => '01',
-             'nethserver-fail2ban-cleanShorewallDynamic' => '10');
+             'nethserver-fail2ban-cleanShorewallDynamic' => '10',
+             'nethserver-fail2ban-remove-obsolete-esmith-F2B-database' => '20');
```
Comment (gsanchietti) [inline createlinks:11]: Again? :(
Please, do not use upper cases in action names.

Argh, I didn't catch `nethserver-fail2ban-cleanShorewallDynamic` before :/

File: root/etc/e-smith/events/actions/nethserver-fail2ban-remove-obsolete-esmith-F2B-database [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see COPYING.
+#
+
+# The old esmith fail2ban database is not used anymore
+# this database is saved inside my.nethesis.it with a subscription
```
Comment (gsanchietti) [inline root/etc/e-smith/events/actions/nethserver-fail2ban-remove-obsolete-esmith-F2B-database:23]: This is true also for subscription:
```
```suggestion
# The old esmith fail2ban database is not used anymore,
# just remove it to not send to subscription server with inventory
```

**Review by gsanchietti — CHANGES_REQUESTED** [review]
Since we already have a cleanup action named `nethserver-fail2ban-clean-sqlite-backup`, let's try to put everything in one place:
- rename `nethserver-fail2ban-clean-sqlite-backup` to something like ``nethserver-fail2ban-cleanup`
- move the code from `nethserver-fail2ban-remove-obsolete-esmith-F2B-database ` to the above action

File: createlinks (hunk 1 above)
Comment (stephdl) [reply createlinks]: done

File: root/etc/e-smith/events/actions/nethserver-fail2ban-remove-obsolete-esmith-F2B-database (hunk 2 above)
Comment (stephdl) [reply root/etc/e-smith/events/actions/nethserver-fail2ban-remove-obsolete-esmith-F2B-database]: done

**Review by gsanchietti — APPROVED** [review]

**Pull request merged**